### ISSUE TITLE
Please open source the app

### ISSUE THREAD

**Issue by b-onc** [mshibanami/redirect-web#15]
By the nature of the extension, it (needs to) has access to all website data. This creates an implicit trust issue.

You can provide more trust by open sourcing the code. That'd speed up development too. And it won't interfere with the subscription model, there are numerous open source iOS apps that are not free on App Store.

**Comment by mshibanami** [comment]
@b-onc Thanks for your feedback. I understand your concern.

As stated in [the privacy policy](https://mshibanami.github.io/redirect-web/#/privacy-policy?id=privacy-policy), this app strictly doesn't collect user data. Well, this is just a legal document, and it technically doesn't address all concerns about potential policy violations. However, even though I made this app OSS, it'd be hard to make it objectively trustworthy because of the deployment. For example, [Dark Reader](https://github.com/darkreader/darkreader) is a great OSS project, and they automate the deployment process, except Safari AFAIK. Therefore, there is still a possibility they put in malicious code. I'm not saying they're fishy. In fact, I use it daily, and I know it's hard to distribute a Safari extension for macOS even without AppStore because of Safari's restrictions. My point is that making the OSS doesn't automatically make the app trustworthy.

> That'd speed up development too.

I have 2 more paid Safari extensions; one used to be OSS for the first few years, and the other is still OSS. Plus, I have a few more OSS projects. Based on these experiences, overall, I've never found that OSS necessarily accelerates development. Of course, it's just my experience, and I know lots of other successful OSSs. But development speed won't be a primary reason to make my apps OSS for me.

> And it won't interfere with the subscription model, there are numerous open source iOS apps that are not free on App Store.

I'm interested in how you're sure it won't interfere. As I said, one of my paid Safari extensions was OSS, but after I changed it to closed source, it became the most downloaded paid macOS app in 10+ countries. This difference makes sense to me because the more paid users, the higher the rank on the App Store, making the app more popular. Since I couldn't do A/B testing for this, this might be ultimately just my imagination. That's why I'm interested in the basis of your idea.

**Comment by mshibanami** [comment]
@b-onc Thanks for your feedback. I understand your concern.

As stated in [the privacy policy](https://mshibanami.github.io/redirect-web/#/privacy-policy?id=privacy-policy), this app strictly doesn't collect user data. Well, this is just a legal document, and it technically doesn't address all concerns about potential policy violations. However, even if I made this app OSS, it'd be hard to make it objectively trustworthy because of the deployment. For example, [Dark Reader](https://github.com/darkreader/darkreader) is a great OSS project, and they automate the deployment process, except Safari AFAIK. Therefore, there is still a possibility they put in malicious code. I'm not saying they're fishy. In fact, I use it daily, and I know it's hard to distribute a Safari extension for macOS even without AppStore because of Safari's restrictions. My point is that making the OSS doesn't automatically make the app trustworthy.

> That'd speed up development too.

I have 2 more paid Safari extensions; one used to be OSS for the first few years, and the other is still OSS. Plus, I have a few more OSS projects. Based on these experiences, overall, I've never found that OSS necessarily accelerates development. Of course, it's just my experience, and I know lots of other successful OSSs. But development speed won't be a primary reason to make my apps OSS for me.

> And it won't interfere with the subscription model, there are numerous open source iOS apps that are not free on App Store.

I'm interested in how you're sure it won't interfere. As I said, one of my paid Safari extensions was OSS, but after I changed it to closed source, it became the most downloaded paid macOS app in 10+ countries. This difference makes sense to me because the more paid users, the higher the rank on the App Store, making the app more popular. Since I couldn't do A/B testing for this, this might be ultimately just my imagination. That's why I'm interested in the basis of your idea.
